libmachine/filestore.go

```diff
@@ -19,7 +19,11 @@ type Filestore struct {
 }
 
 func NewFilestore(rootPath string, caCert string, privateKey string) *Filestore {
-	return &Filestore{path: rootPath, caCertPath: caCert, privateKeyPath: privateKey}
+	return &Filestore{
+		path:           rootPath,
```

Comment:
Side note:  Why do even have a constructor function if all it does it wrap a struct initialization directly?  ;D